MultiplayerMod/Multiplayer/Configuration/GameEventBindings.cs

```diff
@@ -134,6 +137,20 @@ public class GameEventBindings {
 
         DoorEvents.StateChanged += (_, args) => client.Send(new ChangeDoorState(args));
         DoorEvents.OrderUnseal += reference => client.Send(new OrderUnseal(reference));
+
+        AssignableEvents.Assign += (assignable, identity) => {
+            if (identity != null && identity is not KMonoBehaviour) {
```

Comment:
Should not we move this into the command itself?

Because subscription and therefore sending out the command make sense regardless of a type. But it is current command implementation issue that it can not handle non KMonoBehaviour types.

Of course it'll result in sending out non actionable commands (with some arguments being null perhaps).

Reply:
I guess not, I'd rather create a factory, because a command should be a command, not a conversion service, like we decided events should be just events without mp data preparation.
> make sense regardless of a type

Nope, I didn't find any valid thing we need to sync but `KMonoBehavior`.
And I think sending empty commands isn't a good idea, it's maybe ok for now, but I don't see how this is any good.